packages/mobile/src/app/WebViewScreen.tsx

```diff
@@ -37,29 +72,61 @@ function WebViewScreen({ route }: Props) {
     return true
   }
 
-  const hideLoading = () => {
-    setLoading(false)
+  const handleRefresh = () => {
+    webviewRef?.current?.reload()
```

Comment:
nit: `webviewRef` itself can never be null or undefined, right?

Reply:
omg yes you're right, i got confused between the null value of the ref (`webviewRef.current`) and the ref (`webviewRef`) itself 🤦🏻‍♀️ i've removed the redundant null check